lang/cs/Org.Apache.REEF.Network.Examples/GroupCommunication/PipelineBroadcastReduceDriverAndTasks/PipelinedMasterTask.cs

```diff
@@ -63,26 +64,48 @@ public byte[] Call(byte[] memento)
         {
             int[] intArr = new int[_arraySize];
 
+            for (int j = 0; j < _arraySize; j++)
+            {
+                intArr[j] = j;
+            }
+
+            Stopwatch broadcastTime = new Stopwatch();
+            Stopwatch reduceTime = new Stopwatch();
+
             for (int i = 1; i <= _numIters; i++)
             {
-                for (int j = 0; j < _arraySize; j++)
+                intArr[0] = i;
+
+                if (i == 2)
```

Comment:
iteration 1 will require some extra timing issues like sending message to
NameServewr remotely for lookups and reflection time for instantiating
codec and all. Hence, do not want to include the time for 1st iteration.

On Thu, Jul 16, 2015 at 2:36 PM, Julia Wang <notifications@github.com>
wrote:

> In
> lang/cs/Org.Apache.REEF.Network.Examples/GroupCommunication/PipelineBroadcastReduceDriverAndTasks/PipelinedMasterTask.cs
> <https://github.com/apache/incubator-reef/pull/299#discussion_r34840727>:
>
> >              for (int i = 1; i <= _numIters; i++)
> >              {
> > -                for (int j = 0; j < _arraySize; j++)
> > +                intArr[0] = i;
> > +
> > +                if (i == 2)
>
> why reset at 2 not 1?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/apache/incubator-reef/pull/299/files#r34840727>.
>